docs/book/src/user/quick-start.md

```diff
@@ -39,7 +80,7 @@ kubectl create -f {{#releaselink gomodule:"sigs.k8s.io/cluster-api-bootstrap-pro
 
 #### Install Infrastructure Provider Components
 
-{{#tabs name:"tab-installation-infrastructure" tabs:"AWS,vSphere"}}
+{{#tabs name:"tab-installation-infrastructure" tabs:"AWS,Docker,vSphere"}}
```

Comment:
Let's keep Docker last after all cloud providers, wdyt?

Reply:
I chose alphabetical order, because I assume people are used to finding items that way.

Do you want to place Docker last because cloud-backed providers are meant for production, and Docker?

Reply:
Yeah, if that makes sense, we can add a comment to keep non cloud providers at the end

Reply:
I would prefer to keep alphabetical order, because I think newcomers will be able to distinguish the providers by name alone. But I am biased, because I know the various providers. It would be great to have input from newcomers :smile:

Reply:
It's only three of them at the moment.  Let's revisit once we get crowded.